pallets/messages/src/lib.rs

```diff
@@ -58,6 +58,10 @@ pub mod pallet {
 		/// The maximum size of a message [Byte]
 		#[pallet::constant]
 		type MaxMessageSizeInBytes: Get<u32> + Clone;
+
+		/// Retention period for messages [Block]
+		#[pallet::constant]
+		type RetentionPeriodBlocks: Get<u32>;
 	}
```

Comment:
is this temporary? will it be coming from Schema pallet and variable at per schema basis if so we can just define a primitives with some constant value and mark todo for schema pallet with a new issue